.runger-config.yml

```diff
@@ -1,5 +1,5 @@
 expected-num-github-checks: 4
-javascript-watch-command: bin/vite dev
+javascript-watch-command: ./node_modules/.bin/vite --force
```

Comment:
Mostly unrelated change. Related somewhat insofar as I was running this command a lot, and I thought it would be good to update this.